packages/phoenix-evals/src/phoenix/evals/templates.py

```diff
@@ -28,53 +29,111 @@ def get_field(self, field_name: str, args: Sequence[Any], kwargs: Mapping[str, A
         return obj, field_name
 
 
-class PromptTemplate:
+class PromptMessageContentType(str, Enum):
+    TEXT = "text"
+    AUDIO_URL = "audio_url"
```

Comment:
Could this be described by mime types?

Reply:
I know this isn't 1 to 1 but I think the system that needs to evolve is more complicated as a single message can contain multiple multi-modal parts https://cookbook.openai.com/examples/gpt_with_vision_for_video_understanding

Reply:
ah, that's a good point—I'm pretty sure we can handle each "message" or "part" in our template and combine them into a multimodal part in the actual model wrapper. It looks like Anthropic also supports content parts

I'm not sure we need both abstractions (multiple messages + multiple parts) should we rename everything to parts in this PR?